src/manifest.ts

```diff
@@ -53,14 +53,8 @@ export class Manifest {
         return this.manifest['app-id'] || this.manifest.id || 'org.flatpak.Test'
     }
 
-    sdk(): string | null {
-        const sdkPath = this.manifest['build-options']?.['append-path']?.toString()
-        if (sdkPath?.includes('rust')) {
-            return 'rust'
-        } else if (sdkPath?.includes('vala')) {
-            return 'vala'
-        }
-        return null
+    sdkPath(): string | undefined {
```

Comment:
I would create an Sdk type and return it here instead of the append-path. We could map it to the available sdk extensions in #75

Reply:
so we should get it from `manifest['sdk-extensions']`?

Reply:
yes and map it to some "Sdk" type, so that rust-stable/rust-nightly both maps to rust

Reply:
Right I meant in the `Manifest` instead of returning sdkPath we would return an `Sdk`  type that would be something like
`export const type Sdk = 'Vala' | 'Rust' | 'NodeJS' |  'PHP' .............`